package.json

```diff
@@ -51,8 +51,7 @@
   },
   "lint-staged": {
     "*.{js,ts}": [
-      "prettier --log-level warn --ignore-path .gitignore --check",
-      "eslint"
+      "prettier --log-level warn --ignore-path .gitignore --check"
```

Comment:
Why that change, are there fundamental issues with eslint here ?

Reply:
Only a local issue resolved by bumping my node version to a greater minor.
`package.json` has now been set back to its original state.